### PR TITLE
Add faculty members to Sungkyunkwan University (follow-up of #5827)

### PR DESCRIPTION
**Pull request by e-**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts.

- [x] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [x] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

Hello, 

This is a follow-up PR of #5827. 

For the complete list of CS faculty members at SKKU, please refer to the following website: https://sw.skku.edu/eng_sw/index.do.

Recently, our website underwent some changes, and it appears that search engines have not yet updated the new URL (the url starting with http://cs.skku.edu is the old one).

There are three distinct lists available, https://sw.skku.edu/eng_sw/faculty.do, https://sw.skku.edu/eng_sw/faculty_global.do, and https://sw.skku.edu/eng_sw/faculty_intelli_sw.do, with each listing professors in College of Computing and Informatics, who can solely supervise a CS PhD Student).

In addition to the above lists, there are professors affiliated with the "Graduate School of AI" who can also solely supervise CS PhD students. Unfortunately, the list is only available in Korean on the following webpage: https://ai.skku.edu/ai/intro/professor_temp.do. Please allow me to provide you with the English names of the professors in GSAI:

- Eunbyung Park (박은병)
- Hyunjin Park (박현진)
- Jong Hwan Ko (고종환)
- Jungrae Kim (김정래)
- Sukhan Lee (이석한)
- Tae Hee Han (한태희)

All the added faculty members in this PR are either 1) listed on the “three” faculty lists of the College of Computing and Informatics or 2) on the faculty list of GSAI.

I apologize for not including the justification and making you manually search for the information.

### CODE REVIEW

File: csrankings-h.csv [hunk 1]
```diff
@@ -406,10 +406,12 @@ Hau-San Wong,City University of Hong Kong,https://www.cs.cityu.edu.hk/~hswong,i9
 Hausi A. Müller,University of Victoria,http://webhome.cs.uvic.ca/~hausi,8hyNFkYAAAAJ
 Hava T. Siegelmann,University of Massachusetts Amherst,http://binds.cs.umass.edu/havaBio.html,A2fiOI0AAAAJ
 Hayato Yamana,Waseda University,https://www.yama.info.waseda.ac.jp/~yamana/yamana_eng2015.htm,5MXcXrMAAAAJ
+Hayeon Song,Sungkyunkwan University,https://hailab.skku.edu/,BO-AgNYAAAAJ
 Hayfa Zgaya,CRIStAL,https://www.cristal.univ-lille.fr/profil/hzgaya,NOSCHOLARPAGE
 Hayfa Zgaya-Biau,CRIStAL,https://www.cristal.univ-lille.fr/profil/hzgaya,NOSCHOLARPAGE
 Hayley Hung,TU Delft,http://homepage.tudelft.nl/3e2t5,ka-LsrYAAAAJ
 Haym Hirsh,Cornell University,https://www.cs.cornell.edu/~hirsh,lB6jgT4AAAAJ
+Hayoung Oh,Sungkyunkwan University,https://sites.google.com/site/hyoh79,raE3YY0AAAAJ6jgT4AAAAJ
```
Comment (emeryberger) [inline csrankings-h.csv]: Invalid Google Scholar ID.

File: csrankings-s.csv [hunk 2]
```diff
@@ -264,6 +264,7 @@ Sang-Wook Kim,Hanyang University,http://www.agape.hanyang.ac.kr/members/prof,ed2
 Sang-Yoon Chang,UCCS,https://academics.uccs.edu/~schang2,m9j4C0gAAAAJ
 Sang-goo Lee,Seoul National University,http://ids.snu.ac.kr/site/members/M_Sang-goo_Lee.html,sg7xgbsAAAAJ
 Sang-hyo Park,Kyungpook National University,http://sites.google.com/view/knuvi,ZG8REuYAAAAJ
+Sang-Won Lee 0001,Sungkyunkwan University,http://prof.icc.skku.ac.kr/~swlee/,9a-vkpgAAAAJ
```
Comment (emeryberger) [inline csrankings-s.csv:268]: Also needs alias entry for "Sang Won Lee 0001".

File: csrankings-s.csv [hunk 3]
```diff
@@ -2067,6 +2070,7 @@ Sujoy Ghose,IIT Kharagpur,http://www.myopencourses.com/profile/prof-sujoy-ghosh,
 Sujoy Sikdar,Binghamton University,https://www.binghamton.edu/computer-science/contact/profile.html?id=ssikdar,OmZaKTEAAAAJ
 Sujoy Sinha Roy,Graz University of Technology,https://www.iaik.tugraz.at/person/sujoy-sinha-roy,58_wa_4AAAAJ
 Sukhamay Kundu,Louisiana State University,https://www.lsu.edu/eng/cse/people/faculty/kundu.php,NOSCHOLARPAGE
+Sukhan Lee,Sungkyunkwan University,http://isri.skku.ac.kr/,HQ3xdhwAAAAJ
```
Comment (emeryberger) [inline csrankings-s.csv]: Home page unreachable.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by e-** [comment]
Thank you for the review. I have updated what you requested.

File: csrankings-s.csv (hunk 2 above)
Comment (e-) [reply csrankings-s.csv:268]: I have added an alias.

**Comment by emeryberger** [comment]
```
Processing Tamer Abuhmed,Sungkyunkwan University,https://infolab.skku.edu/,pLC4l6YAAAAJ
  Invalid name (Tamer Abuhmed). Please ensure it matches the DBLP entry.
```

(Missing alias)

```Processing Yong Jeong Yi ,Sungkyunkwan University,https://sco.skku.edu/sco/professor/professor.do?mode=view&perId=LZStrE4KQtg6gNgKmBuBLEB3A7ANgIYAUByGA1ACwBGYA5gLxVA%20&,CAC4mjcAAAAJ
  This institution (Sungkyunkwan University) was not found in `institutions.csv`.
  Invalid name (Yong Jeong Yi ). Please ensure it matches the DBLP entry.
```

(spurious space)

**Comment by e-** [comment]
Thank you for the review. I have updated the two entries.

**Pull request merged**